packages/account/src/Sections/Verification/ProofOfIdentity/onfido-sdk-view-container.tsx

```diff
@@ -96,7 +97,11 @@ const OnfidoSdkViewContainer = observer(
                 try {
                     onfido_init.current = await window.Onfido.init({
                         containerId: 'onfido',
-                        language: getOnfidoSupportedLocaleCode(current_language) as SupportedLanguages,
+                        language: {
```

Comment:
why do we have this change in this card?

Reply:
Yes we need to change this because Onfo doesn't support Bengali language currently, we have to pass in the localised phrases to replace the Onfido phrases so that it will be translated.